_posts/2019-12-10-announcing-the-18f-national-security-and-intelligence-portfolio.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "A new focus area for 18F - National Security & Intelligence"
+date: 2019-12-10
+authors:
+- mheadd
+tags:
+- national security
+- portfolios
+excerpt: "18F is launching a new National Security and Intelligence Portfolio to deepen our impact working with agencies working on national
+defense issues."
+Image: /assets/blog/def-intel-portfolio/uncle-sam.jpg
```

Comment:
@Dahianna I think the `I` in Image needs to be lower case in order to process right when the page is rendered.

Reply:
Fixed, thanks!